code-journal.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Life At Ada</title>
+        <link href="styles/main.css" rel="stylesheet">
+    </head>
+    <body>
+
+    </body>
+</html>
```

Comment:
Your secondary pages are empty stubs. I like that they are present because they can help you stay organized.